package.json

```diff
@@ -0,0 +1,107 @@
+{
+  "name": "virumap-app",
+  "version": "0.0.17",
```

Comment:
Should start from 0.0.1

Reply:
https://github.com/virufy/virumap-app/pull/1/commits/27f2aa00ef60acba2f2f14ec4efe9197fa1ff51b